tsa-test/src/test/resources/cell/cell-build/CellBuild.fif

```diff
@@ -21,4 +28,67 @@ PROGRAM{
     ENDC
     HASHCU
   }>
+
+  sdskipfirst PROC:<{
+    5565 PUSHINT // 0b1010110111101, 13 chars
+    NEWC
+    13 STU
+    ENDC
+    CTOS
+    3 PUSHINT
+    SDSKIPFIRST
+    HASHSU
+  }>
+
+  sdsubstr PROC:<{
+    5565 PUSHINT // 0b1010110110101, 13 chars
+    NEWC
+    13 STU
+    ENDC
+    CTOS
+    3 PUSHINT
+    4 PUSHINT
+    SDSUBSTR // 0b0110
+    4 LDU
+    DROP
+  }>
+
+  sdsubstr_bad_offset PROC:<{
+    5565 PUSHINT // 0b1010110110101, 13 chars
+    NEWC
+    13 STU
+    ENDC
+    CTOS
+    1025 PUSHINT // big offset; integer-out-of-range (exit code 5) in sdsubst below
+    4 PUSHINT
+    SDSUBSTR // 0b0110
```

Comment:
Minor: you probably don't need here this comment (copy-pasted from previous test?)